retrofit-adapters/rxjava2/src/main/java/retrofit2/adapter/rxjava2/RxJava2CallAdapterFactory.java

```diff
@@ -59,7 +59,15 @@
    * by default.
    */
   public static RxJava2CallAdapterFactory create() {
-    return new RxJava2CallAdapterFactory(null);
+    return new RxJava2CallAdapterFactory(null, false);
+  }
+
+  /**
+   * Returns an instance which creates asynchronous observables. Applying
```

Comment:
A dispatcher is an implementation detail of OkHttp. All we can guarantee is that `Call.enqueue` is used and I'm not sure that specific information is any more useful than calling them "asynchronous".